src/main/java/gui/SummaryController.java

```diff
@@ -43,13 +43,18 @@ public SummaryController(Collection<Author> inputSummaryData) {
         } catch (IOException e) {
             e.printStackTrace();
         }
-        
+
         ObservableList<AuthorBean> summaryData = FXCollections.observableArrayList();
         for (Author author : inputSummaryData) {
             summaryData.add(new AuthorBean(author));
         }
 
         summaryTable.setItems(summaryData);
+        summaryTable.getSelectionModel().selectedItemProperty()
+                .addListener((observableValue, oldValue, newValue) -> {
```

Comment:
Could you also indent this block to match the previous statement's dot separator?